### DISCUSSION TITLE
Transfer money to my wallet

### DISCUSSION THREAD

**Discussion by Wealker** [schwaaaa/BTCWalletMiner#10] [Q&A]
Good morning.
I have a question:
How can money be transferred? Once you find a wallet, what do you do?

**Comment by mcschwa** [comment]
> Good morning.
> I have a question:
> How can money be transferred? Once you find a wallet, what do you do?

still have an issue with this?

**Reply by snomaen** [reply]
Hey do you think you can give assistance on installing what required system do I need to run

**Reply by mcschwa** [reply]
generally speaking if you just install visual studio with .net 8 you'll be fine

**Reply by snomaen** [reply]
Ok thanks and one more question, amill like to learn more on righting scripts like urs abd grow im certified for pentesting and i wanna grow
